js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
+const fragment = document.createDocumentFragment();
+
+const createPictures = (data, sample) =>{
+  const sampleClone = sample.cloneNode(true);
+  const randomPhoto = sampleClone.querySelector('.picture__img');
+  randomPhoto.srс = data.url;
+  randomPhoto.alt = data.description;
+  const likes = sampleClone.querySelector('.picture__likes');
+  likes.textContent = data.likes;
+  const comments = sampleClone.querySelector('.picture__comments');
+  comments.textContent = data.comments.length;
```

Comment:
Аналогично, но можно и определить переменную избыточную, вроде не базовый критерий
```suggestion
  sampleClone.querySelector('.picture__comments').textContent = data.comments.length;
```

Reply:
+